waiter/src/waiter/util/utils.clj

```diff
@@ -8,7 +8,7 @@
 ;;       The copyright notice above does not evidence any
 ;;       actual or intended publication of such source code.
 ;;
-(ns waiter.utils
+(ns waiter.util.utils
```

Comment:
Nitpick: Not a big fan of the name. It sounds like you're planning on splitting this up soon though, and I don't have a better suggestion right now...